modules/node_modules/@ncigdc/utils/namespace.js

```diff
@@ -4,9 +4,9 @@
 import { compose, withProps, mapProps } from 'recompose';
 
 const namespace = (ns, ...hocs) => compose(
-  withProps(props => ({ [Symbol.for('parentProps')]: props })),
+  withProps(props => ({ $parentProps: props })),
```

Comment:
It was errorring out for reasons unknown, figured the chances of a prop name collision on $parentProps was likely low so decided to go back to normal field name